ibm/service/power/resource_ibm_pi_image.go

```diff
@@ -345,12 +369,39 @@ func resourceIBMPIImageRead(ctx context.Context, d *schema.ResourceData, meta in
 	}
 
 	imageid := *imagedata.ImageID
+	if imagedata.Crn != "" {
+		d.Set(Attr_CRN, imagedata.Crn)
+		tags, err := flex.GetTagsUsingCRN(meta, string(imagedata.Crn))
+		if err != nil {
+			log.Printf("Error on get of ibm pi image (%s) userTags: %s", d.Id(), err)
+		}
+		d.Set(Arg_UserTags, tags)
```

Comment:
```suggestion
		d.Set(Attr_UserTags, tags)
```
This should be `Attr_UserTags` instead.
You will have to add it in the schema as well as in the doc.

Reply:
The argument can be updated now though. As a result it should read from the tagging service since it will return the value the user has in the configuration right.